UICatalog/Scenarios/Wizards.cs

```diff
@@ -217,6 +217,7 @@ void Top_Loaded ()
 						Height = Dim.Fill (1),
 						WordWrap = true,
 						AllowsTab = false,
+						ColorScheme = Colors.Base
```

Comment:
I think changes to Wizard should be in a separate PR.

Reply:
Because it only have the appropriate effect with this PR. But if you insist I'll do a new PR with this PR merged. I'll wait for your feedback.